### ISSUE TITLE
Edit page meta box tabs are not coded accessible tabs

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#360]
<img width="193" alt="Screenshot 2023-11-20 at 9 43 56 AM" src="https://github.com/equalizedigital/accessibility-checker/assets/13153456/95a42d42-2858-452f-9e7d-fd717dfa9afb">

I think these need to be labeled as tabs.
https://www.w3.org/WAI/ARIA/apg/patterns/tabs/examples/tabs-manual/

**Comment by SteveJonesDev** [comment]
@pattonwebz, I just noticed the @amberhinds added the link to the accessible tabs documentation above.

**Comment by amberhinds** [comment]
@pattonwebz, I tested what you have here: https://3.basecamp.com/3579237/buckets/29333825/chats/5333625436@7358576663

Here's some feedback:

role="tablist" is on the wrong element. You have it on the div that contains everything. It should be on the ul.edac-tabs
<img width="1628" alt="Screenshot 2024-05-13 at 10 31 48 PM" src="https://github.com/equalizedigital/accessibility-checker/assets/13153456/a04ce762-a8b7-4943-ba19-5d002d6b63c3">

Otherwise this looks and sound good with screen readers. Thank you!

**Comment by pattonwebz** [comment]
I updated the location of the tablist role and added a very basic screen reader text to use for labeling the tablist. I also tweaked the styles a little so that the buttons look the same as the original links did (there were some unintended visual differences).
